app/views/ops/_diagnostics_orphaned_data_tab.html.haml

```diff
@@ -1,8 +0,0 @@
-- if @sb[:active_tab] == "diagnostics_orphaned_data"
-  - if x_node.split('-').first == "root"
-    = render :partial => "diagnostics_savedreports"
-  - else
-    = render :partial => "layouts/flash_msg"
-    = render :partial => "logs_selected"
```

Comment:
Is this partial still used?

Reply:
Oh nvm - I see it used in the app/views/ops/_diagnostics_collect_logs_tab.html.haml